src/main/java/org/opentripplanner/routing/api/request/preference/MappingPreferences.java

```diff
@@ -0,0 +1,89 @@
+package org.opentripplanner.routing.api.request.preference;
+
+import java.io.Serializable;
+import java.util.EnumSet;
+import java.util.Objects;
+import java.util.Set;
+import java.util.function.Consumer;
+import org.opentripplanner.framework.tostring.ToStringBuilder;
+
+/**
+ * THIS CLASS IS IMMUTABLE AND THREAD-SAFE.
+ */
```

Comment:
Could also add a small description of what this class is.

Reply:
Fixed in e17f2cbc64909d98a216a29bbece71281ad2204f